### ISSUE TITLE
Warnings since upgrade to 1.2

### ISSUE THREAD

**Issue by Sineos** [iobroker-community-adapters/ioBroker.solarlog#20]
Since I upgraded to 1.2 I have regularly following log entries:

```
solarlog.0 | 2019-12-30 16:31:28.846 | warn | (25964) Getforecastdata - Error: TypeError: Cannot read property 'watt_hours_day' of null
solarlog.0 | 2019-12-30 16:31:28.631 | info | (25964) Alle WR/Meter gefunden
solarlog.0 | 2019-12-30 16:31:25.631 | warn | (25964) Nicht alle WR/Meter gefunden
```

SolarLog 300: 4.2.6 Build 115

**Comment by forelleblau** [comment]
Hi
Thanks for your input.
The first two entries:

`solarlog.0 | 2019-12-30 16:31:28.631 | info | (25964) Alle WR/Meter gefunden`
`solarlog.0 | 2019-12-30 16:31:25.631 | warn | (25964) Nicht alle WR/Meter gefunden`

always appear at startup of the adapter (at least the 'info').

Does the 'Getforecastdata - Error' appear only at startup or every hour or occasionally?

Meanwhile I'll check the code...

**Comment by Sineos** [comment]
The error is repeated every hour:

```
solarlog.0 | 2019-12-31 09:25:00.563 | warn | (25964) Getforecastdata - Error: TypeError: Cannot read property 'watt_hours_day' of null
solarlog.0 | 2019-12-31 08:25:00.343 | warn | (25964) Getforecastdata - Error: TypeError: Cannot read property 'watt_hours_day' of null
solarlog.0 | 2019-12-31 07:25:00.364 | warn | (25964) Getforecastdata - Error: TypeError: Cannot read property 'watt_hours_day' of null
```

"Forecast" is not enabled in the adapter config and in SolarLog and no values are entered in respective config items.

**Comment by forelleblau** [comment]
So I have to check why the ,getforecast, - request is submitted. Thanks!

**Comment by forelleblau** [comment]
I found the bug. Just uploaded a fixed version 1.2.1 to github and npm. Hope it will work fine for you.

**Comment by Sineos** [comment]
Works. Many thanks for the support and the great adapter.
